src/templates/static.tsx

```diff
@@ -152,6 +152,8 @@ const StaticPage = ({
   pageContext,
 }: PageProps<Queries.StaticPageQuery, Context>) => {
   const intl = useIntl()
+  // Show sidebar unless 'sidebar: false' frontmatter is present
+  const shouldShowSidebar = mdx.frontmatter.sidebar !== false ? true : false
```

Comment:
Same lines of comment above, I think this should be `shouldShowTableOfContents`.